src/Sylius/Component/Order/Model/OrderItem.php

```diff
@@ -36,6 +36,8 @@ class OrderItem implements OrderItemInterface
      */
     protected $unitPrice = 0;
 
+    protected int $originalUnitPrice = 0;
```

Comment:
With the property, getter and setter not being nullable, wouldn't it break orders made without this feature?

Reply:
Nice catch, thanks! 🚀